doc/user_guide/py_dataframe.md

```diff
@@ -14,7 +14,47 @@ The parameters of `get_dataframe` are the following.
+| object                         | int              | DECIMAL       | all standard and python3       | all         | Uses None for NULL                |
+| object                         | int              | DOUBLE        | all standard and python3       | all         | Uses None for NULL                |
+| object                         | float            | DECIMAL       | all standard and python3       | all         | Uses None or numpy.NaN for NULL   |
+| object                         | float            | DOUBLE        | all standard and python3       | all         | Uses None or numpy.NaN for NULL   |
+| object                         | decimal.Decimal  | DECIMAL       | all standard and python3       | all         | Uses None for NULL                |
+| object                         | decimal.Decimal  | DOUBLE        | all standard and python3       | all         | Uses None for NULL                |
+| object                         | bool             | BOOLEAN       | all standard and python3       | all         | Uses None for NULL                |
+| object                         | str              | (VAR)CHAR     | all standard and python3       | all         | Uses None for NULL                |
+| object                         | pandas.Timestamp | TIMESTAMP     | all standard and python3       | >=6.1.0     | Uses None or pandas.NaN for NULL  |
+| object                         | datetime.date    | DATE          | all standard and python3       | all         | Uses None for NULL                |
+| (u)int*[pyarrow]               | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| (u)int*[pyarrow]               | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| float*[pyarrow]                | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| float*[pyarrow]                | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| string[pyarrow]                | -                | (VAR)CHAR     | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| bool[pyarrow]                  | -                | BOOLEAN       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| decimal128(*)[pyarrow]         | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| decimal128(*)[pyarrow]         | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| timestamp[ns, tz=UTC][pyarrow] | -                | TIMESTAMP     | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support.              |
```

Comment:
Why tz=UTC. In conversion we have:
```
pyTimestamp.reset(PyObject_CallMethod(pyTimestamp.get(), "tz_localize", "z", NULL));
```

Reply:
We can't create a timestamp without a timezone in pandas with pyarrow, but we need a timezone free timestamp for the database, because the database doesn't support timezones.